documentation/developer-guide/modules/tooling-guide/examples/GenerateJsonPayload.java

```diff
@@ -14,13 +14,14 @@
 package examples;
 
 // tag::imports[]
+
+import java.io.File;
+import java.io.IOException;
+
 import org.eclipse.esmf.aspectmodel.generator.json.AspectModelJsonPayloadGenerator;
 import org.eclipse.esmf.aspectmodel.resolver.AspectModelResolver;
 import org.eclipse.esmf.metamodel.Aspect;
 import org.eclipse.esmf.metamodel.loader.AspectModelLoader;
-// end::imports[]
```

Comment:
This is required (at the end of the imports list), because the Antora docs build uses it to select parts of the example file